projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.html

```diff
@@ -1,73 +1,67 @@
 <div class="external-feed page-padding">
-  <div class="page-container">
-    <div class="feed-desktop d-flex justify-content-start">
-      <div class="left-column">
-        <nb-card class="tag-checkboxes">
-          <div class="header">
-          Popular Tags
-          </div>
-          <div class="tag-checkbox" *ngFor="let tag of tags">
-            <input
-              type="checkbox"
-              name="tags"
-              value="{{ tag }}"
-              [checked]="false"
-              (change)="updateCheckedTags(tag, $event)"
-            />
-            <label for="#{{ tag }}"> #{{ tag }} </label>
-            <br />
-            <br />
-          </div>
-        </nb-card>
-      </div>
-      <div class="right-column">
-        <nb-card class="header">
-          <nb-card-body>
-            <h1>
-              <nb-icon icon="cast-outline" status="warning"></nb-icon>
-              Feed
-            </h1>
-            <p>Some links from around the world!</p>
-          </nb-card-body>
-        </nb-card>
-        
-        <nb-card>
+  <div class="page-container d-flex justify-content-center">
+    <div class="left-column">
+      <nb-card class="tag-checkboxes">
+        <div class="header">Popular Tags</div>
+        <div class="tag-checkbox" *ngFor="let tag of tags">
+          <input
+            type="checkbox"
+            name="tags"
+            value="{{ tag }}"
+            id="#{{ tag }}"
+            [checked]="false"
+            (change)="updateCheckedTags(tag, $event)"
+          />
+          <label for="#{{ tag }}"> #{{ tag }} </label>
+          <br />
+          <br />
+        </div>
+      </nb-card>
+    </div>
+    <div class="right-column">
+      <nb-card class="header">
+        <nb-card-body>
+          <h1>
+            <nb-icon icon="cast-outline" status="warning"></nb-icon>
+            Feed
+          </h1>
+          <p>Some links from around the world!</p>
+        </nb-card-body>
+      </nb-card>
+
+      <nb-card>
         <div class="sorting d-flex justify-content-center">
-          <button
-          (click)="getLatestPosts()">
+          <button (click)="getLatestPosts()">
             <nb-icon icon="sun" status="warning"></nb-icon>
             Latest
           </button>
-          <button
-          (click)="getPopularPosts()">
+          <button (click)="getPopularPosts()">
             <nb-icon icon="flash" status="warning"></nb-icon>
             Hot
           </button>
-          <hr>
+          <hr />
```

Comment:
Maybe we can remove this? What do you think?

Reply:
Removed